app/src/main/java/org/wildstang/wildrank/androidv2/activities/SyncActivity.java

```diff
@@ -50,10 +60,37 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     private void beginSync() {
-        if (!SyncUtilities.isFlashDriveConnected()) {
-            showDriveNotMountedWarning();
+        if (SyncUtilities.useSAF()) {
+            SyncUtilities.openFileChooser(this);
```

Comment:
Awesome. My team was using KitKat when I was last there and I don't think that's changed, I'll have them test it out soon(tm).